packages/modules/src/modules/banners/momentTemplate/stories/MomentTemplateBanner.stories.tsx

```diff
@@ -61,12 +61,15 @@ Default.args = {
             },
         },
     },
-    numArticles: 50,
+    articleCounts: {
+        for52Weeks: 50,
+        forTargetedWeeks: 50,
+    },
     tickerSettings: undefined,
 };
 
 export const WithBackDrop = WithBackDropTemplate.bind({});
-Default.args = {
+WithBackDrop.args = {
```

Comment:
This was using the wrong data of Default for WithBackDrop , hence corrected .This caused changes in three UI tests in the checks